mabwiser/mab.py

```diff
@@ -964,11 +964,38 @@ def add_arm(self, arm: Arm, binarizer: Callable = None, scaler: Callable = None)
                        TypeError("Scaler must be a scaler object from sklearn.preprocessing with a transform method"))
             check_true(hasattr(scaler, 'mean_') and hasattr(scaler, 'var_'),
                        TypeError("Scaler must be fit with calculated mean_ and var_ attributes"))
+        check_false(arm in self.arms, ValueError("The arm is already in the list of arms."))
 
         self._validate_arm(arm)
         self.arms.append(arm)
         self._imp.add_arm(arm, binarizer, scaler)
 
+    def remove_arm(self, arm: Arm) -> NoReturn:
+        """Removes an _arm_ from the list of arms.
+
+        Parameters
+        ----------
+        arm: Arm
+            The existing arm to be removed.
+
+        Returns
+        -------
+        No return.
+
+        Raises
+        ------
+        ValueError: The arm does not exist.
+        ValueError: The arm is ``None``.
+        ValueError: The arm is ``NaN``.
+        ValueError: The arm is ``Infinity``.
+        """
+
+        check_true(arm in self.arms, ValueError("The arm is not in the list of arms."))
```

Comment:
I removed the "check arm is in" and the "check arm is not in" from `self._validate_arm` so that I can use it for both `add_arm` and `remove_arm`. Alternatively, I would have had to use separate validation methods for add and remove.